features/get-dataset-json.feature

```diff
@@ -1116,7 +1116,7 @@ Feature: Get Dataset JSON
     {
       "request": {
         "dimension":       "city",
-        "populationType":  "Example",  
```

Comment:
This change does not seem to be coming from `master` as this scenario does not exist in master. Not sure how it's ended up here if this PR is just to bring develop and master in line?